docker/Dockerfile

```diff
@@ -75,11 +75,20 @@ RUN sed -i -r "s|(base_url:)(\s+.+)|base_url: http://app.catroweb/index_test.php
 # Overwrite project config:
 ARG APP_ENVIRONMENT
 RUN echo "\n" >> .env.local && \
-    echo APP_ENV=$APP_ENVIRONMENT >> .env.local && \
+    printf "%s\n" APP_ENV=$APP_ENVIRONMENT \
+     ELASTICSEARCH_URL=http://localhost:9200/ \
+     ES_HOST=localhost \
+     ES_PORT=9200 >> .env.local && \
     echo "\n" >> .env.dev.local && \
-    echo DATABASE_URL=pdo-mysql://root:root@db.catroweb.dev:3306/catroweb_dev >> .env.dev.local && \
+    printf "%s\n" DATABASE_URL=pdo-mysql://root:root@db.catroweb.dev:3306/catroweb_dev \
```

Comment:
if it is the same config. no need to overwrite it twice, just put it into the local file.

+ I'm sure the URL and host is wrong